app/classifier/annotation-renderer/svg.jsx

```diff
@@ -153,7 +153,17 @@ export default class SVGRenderer extends React.Component {
       })
       .filter(Boolean);
     children = children.concat(persistentHooks);
-
+    const childrenWithViewBox = React.Children.map(
+      this.props.children,
+      child => React.cloneElement(
+        child,
+        type === 'canvas' ? {
+          viewBoxDimensions: this.props.viewBoxDimensions,
+          classification: this.props.classification,
+          subject: this.props.subject,
+          workflow: this.props.workflow
+        } : {})
+    );
```

Comment:
This block introduces a strong coupling between drawing tasks and the canvas viewer.

Reply:
moved the injection of these props to the frame-viewer, with viewBoxDimensions being added in the frame-annotator and a default case added in canvas-viewer